morebits.js

```diff
@@ -1374,18 +1376,32 @@ Morebits.wiki.api.prototype = {
 	query: null,
 	response: null,
 	responseXML: null,  // use `response` instead; retained for backwards compatibility
-	setParent: function(parent) {
-		this.parent = parent;
-	},  // keep track of parent object for callbacks
 	statelem: null,  // this non-standard name kept for backwards compatibility
 	statusText: null, // result received from the API, normally "success" or "error"
 	errorCode: null, // short text error code, if any, as documented in the MediaWiki API
 	errorText: null, // full error description, if any
 
+	/**
+	 * Keep track of parent object for callbacks
+	 * @param {*} parent
+     */
```

Comment:
Inline with #780, shouldn't this be on one line? (same applies to `setStatusElement` below)

Reply:
Done for the latter. Here there are two lines - one for description and one for param. While I guess putting them on the same line is still valid JSDoc, it looks clumsy.